src/components/PokemonTile.js

```diff
@@ -0,0 +1,64 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+
+function PokemonTile({name, endpoint}) {
+    const [singlePokemonData, setSinglePokemonData] = useState({});
```

Comment:
Als je hier voor de default waarde `undefined` gaat, dan kan je check verderop iets simpeler
```suggestion
    const [singlePokemonData, setSinglePokemonData] = useState();
```

Reply:
Ook als ik dit wijzig, krijg ik de volgende foutmelding: 
Cannot destructure property 'weight' of 'singlePokemonData' as it is undefined.

Dus ik houd het even op ({})